### PR TITLE
Show armor 1 exotics in exotic picker.

### LINKED ISSUE #6815: Find armor 1.0 exotics in the exotics picker (bring back armor 1.0 to LO)
You don't have to calculate the stats if you don't want to but we should probably allow LO to "find" armor 1.0 exotics so people stop getting confused. You could display them in red with a warning or something, but don't hide them.

### PR DESCRIPTION
**Pull request by ryan-rushton**
Screen Shot 2021-06-19 at 11 06 55 am

Closes #6815

### CODE REVIEW

File: src/app/loadout-builder/LoadoutBuilder.tsx [hunk 1]
```diff
@@ -101,6 +103,33 @@ function mapStateToProps() {
     }
   );
 
+  const unusableExoticsSelector = createSelector(
```
Comment (ryan-rushton) [inline src/app/loadout-builder/LoadoutBuilder.tsx:106]: Was tempted to build this into the items selector so we only loop over all items once. This way separates concerns but loops twice and the logic is very similar.

File: src/locale/dim.json [hunk 2]
```diff
@@ -374,6 +374,7 @@
     },
     "HideAllConfigs": "Hide all configurations",
     "HideConfigs": "Hide configurations",
+    "IncompatibleWithOptimizer": "This item is incompatible with the optimizer. Please purchase a new version from collections.",
```
Comment (nev-r) [inline src/locale/dim.json:377]: Optimizer & Collections should be capitalized, and the game/help consistently calls it "reacquire"

**Review by nev-r — COMMENTED** [review]
image

is this the intention?

File: src/locale/dim.json (hunk 2 above)
Comment (ryan-rushton) [reply src/locale/dim.json:377]: Ok will update

**Comment by ryan-rushton** [comment]
No it wasn't, I don't have any armor 1 and didn't realise they have different hashes. Using a name comparison now instead.

File: src/locale/dim.json (hunk 2 above)
Comment (ryan-rushton) [reply src/locale/dim.json]: Done

**Review by nev-r — APPROVED** [review]

**Pull request merged**